playbooks/adhoc/uninstall.yml

```diff
@@ -125,13 +125,13 @@
       failed_when: False
       with_items: "{{ exited_containers_to_delete.results }}"
 
-    - shell: docker images | grep {{ item }} | awk '{ print $3 }'
+    - shell: docker images | egrep {{ item }} | awk '{ print $3 }'
       changed_when: False
       failed_when: False
       register: images_to_delete
       with_items:
-        - registry.access.redhat.com/openshift3
-        - registry.access.redhat.com/aep3
+        - registry\.access\.*redhat\.com/openshift3
+        - registry\.access\..*redhat\.com/aep3
```

Comment:
the double dots were on purpose.  I wanted to escape all the literal dots but then allow for `.*` to match anything in between.  Looks like I didn't do the same for the openshift3 case. :/